protocol/communities/community.go

```diff
@@ -233,6 +237,10 @@ func (o *Community) MarshalJSON() ([]byte, error) {
 	}{
 		ID:                          o.ID(),
 		MemberRole:                  o.MemberRole(o.MemberIdentity()),
+		IsAdmin:                     o.IsAdmin(),
+		IsOwner:                     o.IsOwner(),
+		IsControlNode:               o.IsControlNode(),
+		HasAdminPrivileges:          o.IsOwnerOrAdmin(),
```

Comment:
The properties `IsAdmin` and `IsOwner` are mutually exclusive and are already better(imo) represented by the `MemberRole`.

Reply:
+1, MemberRole will be enough
P.S. IsControlNode is ok

Reply:
returned back!